codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/utils/IOUtils.java

```diff
@@ -69,10 +69,9 @@ public static void writeTokenTreesIntoDir(
   /**
    * Evaluate a simple template as a resource under "/templates/<templatePath>"
    * and then write it to "<rootPath>/<templateOutputPath>".
-   * The template output path is also evaluated as in {@link #safeEvalPathTemplate(String, Map)}
```

Comment:
This comment change isn't accurate, as it concerns the template output path and not the template resource path.